sites/automationworld/server/components/document.marko

```diff
@@ -16,6 +16,7 @@ $ const { site } = out.global;
     <marko-web-gam-enable />
   </@head>
   <@above-container>
+    <marko-web-browser-component name="TrackLeaders" />
```

Comment:
Make a wrapping marko component for this:  `<common-gtm-track-leaders />` that calls:
`<marko-web-browser-component name="CommonGTMTrackLeaders" />`